subt_ign/src/CommsBrokerPlugin.cc

```diff
@@ -310,7 +311,9 @@ void CommsBrokerPlugin::UpdateIfNewBreadcrumbs()
   for (const auto& [name, pose] : this->poses)
   {
     // New breadcrumb found.
-    if (name.find("__breadcrumb__") != std::string::npos &&
+    // A static model is spawned when the breadcrumb is made static
+    if (std::regex_match(name,
```

Comment:
Confirming for documentation purposes that the breadcrumb will only affect communication after the `<disable_physics_time>`, which is currently 3 seconds, correct?

Reply:
yes that's correct